products/memgraph/upcoming/tutorials/marvel-universe.md

```diff
@@ -0,0 +1,165 @@
+## Marvel Universe Social Network
```

Comment:
You should maybe add `Comic` to the title because in the text you refer to `MCU`, not `MU`.

Reply:
Sure, will do.